### DISCUSSION TITLE
Support for ST75160 display

### DISCUSSION THREAD

**Discussion by johngineer** [olikraus/u8g2#1641] [General]
Hello! I am working with a 160x96 monochrome display that uses an ST75160 controller. It would be great if you could add support for this chip to u8g2.

To that end, I am willing to provide a sample display + breakout PCB for test & eval. I just need to know where to send it.

Thanks!

**Comment by olikraus** [comment]
Created an issue for this: https://github.com/olikraus/u8g2/issues/1642

**Reply by TejcaSrejca** [reply]
Hello! What do I need to change in this library  U8G2_ST75160_JM16096_1_4W_SW_SPI that I could use it for 128x64 resolution?
My pins are a little different. I have SDA, SCL, CS, A0 and reset.

**Reply by olikraus** [reply]
A new issue would be nice for further tracking. 
The current  U8G2_ST75160_JM16096_1_4W_SW_SPI was never tested, nevertheless, maybe you can try to run this constructor in SW SPI mode, Will be something visible? If so, post a picture into the new issue.
